interactive/components/details/Rule.vue

```diff
@@ -60,6 +60,15 @@ function getRegexperLink(regex: RegExp) {
 function getCsGitHubLink(key: RegExp | string, repo = 'unocss/unocss') {
   return `https://cs.github.com/?scope=${encodeURI(`repo:${repo}`)}&q="${encodeURIComponent(String(key))}"`
 }
+
+function copyClassName(className: string) {
```

Comment:
How about use [useclipboard](https://vueuse.org/core/useclipboard/)?

Reply:
Thanks for your advice